File: code/1048/1048.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const salary = parseFloat(lines[0]);

let percente;

if (salary <= 400.0) {
  percente = 15.0;
} else if (salary <= 800.0) {
  percente = 12.0;
} else if (salary <= 1200.0) {
  percente = 10.0;
} else if (salary <= 2000.0) {
  percente = 7.0;
} else {
  percente = 4.0;
}

console.log(`Novo salario: ${(salary * (1 + percente / 100)).toFixed(2)}`);
console.log(`Reajuste ganho: ${(salary * (percente / 100)).toFixed(2)}`);
console.log(`Em percentual: ${percente.toFixed(0)} %`);

File: code/1048/1048.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        decimal salary;

        decimal.TryParse(Console.ReadLine(), out salary);

        if (salary <= 400)
        {
            Print(salary, 15);
        }
        else if (salary <= 800)
        {
            Print(salary, 12);
        }
        else if (salary <= 1200)
        {
            Print(salary, 10);
        }
        else if (salary <= 2000)
        {
            Print(salary, 7);
        }
        else
        {
            Print(salary, 4);
        }
    }

    private static void Print(decimal salary, int percent)
    {
        decimal new_salary = salary * (1 + percent / 100m);
        decimal money_earned = salary * percent / 100m;

        Console.WriteLine($"Novo salario: {new_salary:0.00}");
        Console.WriteLine($"Reajuste ganho: {money_earned:0.00}");
        Console.WriteLine($"Em percentual: {percent} %");
    }
}